Servers/utils/assessment.utils.ts

```
@@ -3,36 +3,44 @@ import pool from "../database/db";
+  projectId: number
+): Promise<Assessment[]> => {
+  const result = await pool.query(
+    "SELECT * FROM assessments WHERE project_id = $1",
+    [projectId]
+  );
+  return result.rows;
+};
+
+export const createNewAssessmentQuery = async (
+  assessment: Assessment
+): Promise<Object> => {
```

Biome (1.9.4)

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)